### DISCUSSION TITLE
how to define the width of spiral_double

### DISCUSSION THREAD

**Discussion by luzhoumingyue** [gdsfactory/gdsfactory#1047] [Q&A] [answered]
hi sirs,i am going to learn gdsfactory,and noticed spiral_double,may i know how to define the width of spiral_double?many thanks

**Comment by simbilod** [comment, marked as answer]
Welcome!

If you look at the [docs](https://gdsfactory.github.io/gdsfactory/components.html#spiral-double) for spiral_double, you see that the spiral is made from the CrossSection that you give it in the cross_section argument

There are predefined cross_sections in [gdsfactory.cross_section](https://gdsfactory.github.io/gdsfactory/_modules/gdsfactory/cross_section.html#CrossSection), for instance `strip` that has a width argument. You can also make your own.

Feel free to share your code here for us to look at!

**Reply by luzhoumingyue** [reply]
hi simbilod sir,thank u for ur great support,and i will have a look about cross_section,many thanks~
